test/Rings-test.jl

```diff
@@ -6,8 +6,8 @@ const ring_to_mat = Dict(ZZ                         => ZZMatrix,
                          finite_field(ZZRingElem(3), 2, "b")[1] => FqPolyRepMatrix,
                          ArbField()                      => ArbMatrix,
                          AcbField()                      => AcbMatrix,
-                         RealField()                     => RealMat,
-                         ComplexField()                  => ComplexMat,
+                         RealField()                     => RealMatrix,
+                         ComplexField()                  => ComplexMatrix,
```

Comment:
Independent of this PR: can't we replace `ring_to_mat` by suitable use of `dense_matrix_type`?

Reply:
Yeah, should be possible